src/img2catalog/cli_app.py

```diff
@@ -249,36 +249,36 @@ def output_dcat(ctx: click.Context, output: click.Path, format: str):
         print(g.serialize(format=format))
 
 
-@click.option("-f", "--fdp", envvar='XNATDCAT_FDP', type=str, required=True, help="URL of FDP to push to")
+@click.option("-f", "--fdp", envvar="img2catalog_FDP", type=str, required=True, help="URL of FDP to push to")
 @click.option(
-    "-u", "--username", envvar='XNATDCAT_FDP_USER', type=str, required=True, help="Username of FDP to push to"
+    "-u", "--username", envvar="img2catalog_FDP_USER", type=str, required=True, help="Username of FDP to push to"
 )
 @click.option(
-    "-p", "--password", envvar='XNATDCAT_FDP_PASS', type=str, required=True, help="Password of FDP to push to"
+    "-p", "--password", envvar="img2catalog_FDP_PASS", type=str, required=True, help="Password of FDP to push to"
```

Comment:
I would expect environment variables to be uppercase. This is probably a side effect of a general find-and-replace